http/sentryhttp_test.go

```diff
@@ -33,6 +33,7 @@ func TestIntegration(t *testing.T) {
 			}),
 
 			WantEvent: &sentry.Event{
+				Release: sentry.GetDefaultRelease(),
```

Comment:
For the tests that live outside of `package sentry`, we don't really need to test the behavior of getting a default release string.

```suggestion
```

We can remove again the `Release` field from the tests, and explicitly ignore it in the [comparison options down below](https://github.com/getsentry/sentry-go/pull/363/files#diff-f5c2234d1ccf4d9b66809ca2e4c7a9e1a29b0e3742b110222ab843a896359fa0R219-R223).

```go
cmpopts.IgnoreFields(
	sentry.Event{},
	"Contexts", "EventID", "Extra", "Platform", "Release",       // <--- added "Release" here
	"Sdk", "ServerName", "Tags", "Timestamp",
),
```